src/server.jl

```diff
@@ -12,10 +12,10 @@ immutable MaximaSession <: Base.AbstractPipe
     output::Pipe
     process::Base.Process
 
-    function MaximaSession()
+    function MaximaSession(;ArgString="")
```

Comment:
Camel case is typically reserved for module names or types. `argstring` or just `args` might be more appropriate as
its fairly clear what this refers to in the case of a `MaximaSession`.